console/src/main/java/org/eclipse/rdf4j/console/ConsoleIO.java

```diff
@@ -61,6 +61,14 @@ public ConsoleIO(ConsoleState info) throws IOException {
 		this.appInfo = info;
 	}
 
+	/**
+	 * 
+	 * @return 
+	 */
+	public OutputStream getOutputStream() {
+		return terminal.output();
```

Comment:
Does this output stream get closed? Or should it maybe be configured as close-on-exit?

Reply:
Should be closed by the underlying (JLine) terminal on exit, but it's worth to double-check...

Reply:
I'll create a separate issue for explicitly closing the JLine terminal (currently - including previous versions of the Console - the output stream just stays open until the console exits with System.exit. But it won't harm to explicitly invoke the close() method on the terminal when exiting the Console)